### ISSUE TITLE
NVDA and firefox errors

### ISSUE THREAD

**Issue by joshknnd1982** [nvaccess/nvda#6496]
When I go to certain websites from the bookmarks toolbar I get certain error sounds with this try build. The error log is attached. 
[nvda_error_log.txt](https://github.com/nvaccess/nvda/files/546458/nvda_error_log.txt)

**Comment by jcsteh** [comment]
Do you notice a problem that affects you when this occurs or do you just hear the error sound?

In short, this exception can occur because NVDA is querying an editable text field, but the editable text field disappears as NVDA is trying to query it. It should be harmless (since the field has disappeared anyway, so no text can be retrieved). Ideally, we'd catch this and log it as a debugWarning, but I'm not quite sure exactly where we'd catch it. I'm reluctant to just wrap that whole block in a try/except, since that might hide other problems.

P3 because we should ideally clean this up, but it's low impact, especially in release builds where the error sound is disabled.

**Comment by Adriani90** [comment]
@joshknnd1982 are you still having this issue in Firefox 65.0.2 and NVDA 2019.1 RC1?

**Comment by joshknnd1982** [comment]
No. I am no longer having this issue.



Sent from Mail for Windows 10

**Comment by Adriani90** [comment]
Thanks very much for reporting back. Closing as works for me.
